workers/tasks/export/html.js

```diff
@@ -0,0 +1,250 @@
+					}
+					return null;
+				},
+			})
+			.css.toString()
+			// Find all things like url(fonts/KaTeX_whatever) and replace them with a version that
+			// is loaded from an external CDN.
+			.replace(
+				/url\((fonts\/KaTeX_(?:[A-z0-9\-_]*?).(?:[A-z0-9]+))\)/g,
+				(_, fontPath) => `url(${katexCdnPrefix + fontPath})`,
+			);
+		fs.writeFileSync(cssPath, css);
+	}
+	return fs.readFileSync(cssPath).toString();
+};
+
+const staticCss = createCss();
+
+const filterNonExportableNodes = (nodes) =>
+	nodes.filter((n) => !nonExportableNodeTypes.includes(n.type));
+
+const addAttrsToNodes = (newAttrs, matchNodeTypes, nodes) =>
+	nodes.map((node) => {
+		if (matchNodeTypes.includes(node.type)) {
+			return {
+				...node,
+				attrs: {
+					...node.attrs,
+					...(typeof newAttrs === 'function' ? newAttrs(node) || {} : newAttrs),
+				},
+			};
+		}
+		if (node.content) {
+			return {
+				...node,
+				content: addAttrsToNodes(newAttrs, matchNodeTypes, node.content),
+			};
+		}
+		return node;
+	});
+
+const renderEquationsAsScripts = (nodes) =>
+	addAttrsToNodes({ renderForPandoc: true }, ['equation', 'block_equation'], nodes);
+
+const addHrefsToNotes = (nodes) =>
+	addAttrsToNodes(
+		(node) => {
+			const { count } = node.attrs;
+			if (node.type === 'citation') {
+				return {
+					href: `#${citationPrefix}-${count}`,
+					id: `${citationPrefix}-${count}-return`,
+				};
+			}
+			if (node.type === 'footnote') {
+				return {
+					href: `#${footnotePrefix}-${count}`,
+					id: `${footnotePrefix}-${count}-return`,
+				};
+			}
+			return {};
+		},
+		['citation', 'footnote'],
+		nodes,
+	);
+
+const blankIframes = (nodes) => addAttrsToNodes({ url: 'about:blank' }, ['iframe'], nodes);
+
+const renderFrontMatterForPandoc = ({ updatedDateString, publishedDateString, doi }) => {
+	const showUpdatedDate = updatedDateString && updatedDateString !== publishedDateString;
+	return (
+		<React.Fragment>
+			{showUpdatedDate && (
+				<div>
+					<strong>Updated on:</strong> {updatedDateString}
+				</div>
+			)}
+			{doi && (
+				<div>
+					<strong>DOI:</strong> {doi}
+				</div>
+			)}
+		</React.Fragment>
+	);
+};
+
+const renderFrontMatterForHtml = ({
+	updatedDateString,
+	publishedDateString,
+	primaryCollectionTitle,
+	doi,
+	title,
+	communityTitle,
+	attributions,
+}) => {
+	const showUpdatedDate = updatedDateString && updatedDateString !== publishedDateString;
+	const affiliations = [
+		...new Set(attributions.map((attr) => attr.affiliation).filter((x) => x)),
+	];
+	return (
+		<section className="cover">
+			<h3>
+				<span className="community">{communityTitle}</span>
+				{primaryCollectionTitle && (
+					<span className="collection">
+						{bullet}
+						{primaryCollectionTitle}
+					</span>
+				)}
+			</h3>
+			<h1 className="title">{title}</h1>
+			{attributions.length > 0 && (
+				<div className="byline">
+					<h3>
+						{attributions.map((attr, index) => {
```

Comment:
This needs to filter for `isAuthor`. Currently, it is displaying all contributors on the title page.

Reply:
👍